File: server/src/main/java/com/fluffy/comment/application/dto/AuthorResponse.java
package com.fluffy.comment.application.dto;

import com.fluffy.auth.domain.Member;

public record AuthorResponse(
        Long id,
        String name,
        String avatarUrl
) {

    public static AuthorResponse from(Member member) {
        return new AuthorResponse(
                member.getId(),
                member.getName(),
                member.getAvatarUrl()
        );
    }
}

File: server/src/main/kotlin/com/fluffy/comment/application/response/AuthorResponse.kt
package com.fluffy.comment.application.response

import com.fluffy.auth.domain.Member

data class AuthorResponse(
    val id: Long,
    val name: String,
    val avatarUrl: String,
) {

    companion object {

        fun from(member: Member): AuthorResponse {
            return AuthorResponse(
                member.id,
                member.name,
                member.avatarUrl,
            )
        }
    }
}
